src/portal/src/polyfills.ts

```diff
@@ -37,3 +37,6 @@ import 'intl/locale-data/jsonp/zh';
 
 import 'zone.js/dist/zone';
 
+(window as any).global = window;
```

Comment:
why assign `window` to its global property?

Reply:
global is a node js window.